README.rst

```diff
@@ -72,3 +72,10 @@ Questions can also be asked on
 
 .. image:: https://github.com/rwth-i6/returnn/workflows/CI/badge.svg
     :target: https://github.com/rwth-i6/returnn/actions
+
+Dependencies
+============
+
+pip dependencies are listed in ``requirements.txt`` and ``requirements-dev``, although some parts of the code may require additional dependencies (e.g. ``librosa``, ``resampy``) on-demand.
```

Comment:
```suggestion
Pip dependencies are listed in ``requirements.txt`` and ``requirements-dev``,
although some parts of the code may require additional dependencies (e.g. ``librosa``, ``resampy``) on-demand.

For TensorFlow-based setups, RETURNN requires TF >=2.

For PyTorch-based setups, RETURNN requires Torch >=1.
```
(I actually don't really know our min required Torch version... But this here would be some good starting point.)